packages/api-client/src/components/ImportCollection/ImportCollectionModal.vue

```diff
@@ -146,78 +145,87 @@ onUnmounted(() => {
       <div
         class="flex items-center flex-col m-auto px-8 py-8 rounded-xl border-1/2 max-w-[380px] w-full transition-opacity"
         :class="{ 'opacity-0': prefetchResult.state === 'loading' }">
-        <!-- Logo -->
-        <IntegrationLogo
-          v-if="shouldShowIntegrationIcon"
-          :integration="integration" />
-        <!-- Title -->
-        <div class="text-center text-md font-bold mb-2 line-clamp-1">
-          {{ title ?? 'Untitled Collection' }}
-        </div>
-        <div class="text-c-1 text-sm font-medium mb-2 text-center text-balance">
-          Import the OpenAPI document to instantly send API requests. No signup
-          required.
-        </div>
-        <!-- Prefetch error -->
-        <PrefetchError
-          v-if="prefetchResult.error"
-          :result="prefetchResult" />
-        <!-- Actions -->
-        <template v-else-if="version">
-          <div class="inline-flex flex-col gap-2 items-center z-10 w-full">
-            <!-- <OpenAppButton :source="source" /> -->
-            <ImportNowButton
-              :source="prefetchResult?.url ?? source"
-              variant="button"
-              :watchMode="watchMode"
-              @importFinished="() => $emit('importFinished')" />
+        <template v-if="prefetchResult.error">
+          <!-- Heading -->
+          <div class="text-center text-md font-bold mb-2 line-clamp-1">
+            Missing OpenAPI document
```

Comment:
What do you think about

```suggestion
            No OpenAPI document found
```

Reply:
<img width="432" alt="Screenshot 2024-10-30 at 14 30 25" src="https://github.com/user-attachments/assets/c0060432-41ba-4d41-94fc-029c5b27cc7d">